### PR TITLE
fix: reexport properly

### PR DESCRIPTION
**Pull request by mattsse**
use `import "@foundry-rs/hardhat-forge";`

tested with local project `yard add file:...`

### CODE REVIEW

**Comment by guacamoleoverflow** [comment]
apologies for the random comment - I think some of this dist stuff is just related to how `tsc` builds to the output folder? I'm not that familiar myself but the current `dist` is including src/test, and I'm not sure we'd want that in npm? Usually you just include the compiled src to make the download a little smaller. I think we'd just need to add `include` the other `tsconfig` files?

for example: 

for `hardhat-forge`, 

image

the `dist` folder includes the `test` folder, also the `src` folder. but with `"include": ["src/**/*.ts"]`

image

there's only the contents of the `src` folder, and you can just specify `dist/index.js` for `"main"` in `package.json`

(I was testing this myself the other day and stumbled upon #26/#31 earlier, maybe next time I'll do a PR 😄 )

File: packages/hardhat/src/index.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import "../../hardhat-forge/src/index";
-import "../../hardhat-anvil/src/index";
+import "@foundry-rs/hardhat-forge";
```
Comment (tynes) [inline packages/hardhat/src/index.ts:1]: Does this actually work? It looks like its just importing and not exporting.

Google search says you can do something like:
```ts
export * from "@foundry-rs/hardhat-forge";
```

Comment (guacamoleoverflow) [reply packages/hardhat/src/index.ts:1]: Yeah if you want to export, you'd want that. Can also do `export * as forge from "@foundry-rs/hardhat-forge";` if you want the exports to have a name vs import each thing from it.

the compiled version currently is the equivalent of just the imports, so

```js
require("@foundry-rs/hardhat-forge");
require("@foundry-rs/hardhat-anvil");
```

Comment (mattsse) [reply packages/hardhat/src/index.ts]: nice, using aliases now

**Pull request merged**